tests/test_dataset.py

```diff
@@ -73,7 +73,10 @@ def test_spatial_dataset(self):
             model_kwargs = json.load(file)
 
         model = Presto.construct(**model_kwargs)
+        # valid_month token can only be used in finetuning model
+        model = model.construct_finetuning_model(num_outputs=1)
         model.to(device)
+        model.encoder.valid_month_as_token = True
         eo, dw, mask, latlons, months, _, valid_months, _, _ = ds[0]
 
         # TODO: investigate why inference test fails with valid_month
```

Comment:
@cbutsko tests no longer fail, TODO can be removed?

Reply:
yes, this is fixed